docs/provider/daemon/getting_started/run_remote.md

```diff
@@ -0,0 +1,164 @@
+
+To secure the connection with the Trade interface of your daemon, depending on your needs, you can opt for TLS encryption or spinnig up a Onion hidden service.
+
+### Enable TLS for Trade interface
+
+It is possible to enable TLS encryption and requiring to establish a secure connection with the public Trade interface of the daemon.
+
+For this, you must obtain a valid TLS certificate from a trusted Certificate Authotity (for example [Let's Encrypt](https://letsencrypt.org/), [ZeroSSL](https://zerossl.com/), etc.) and export the `TDEX_SSL_KEY` and `TDEX_SSL_CERT` environment variables with the paths of your TLS key and cert files:
+
+- Standalone:
+    ```sh
+    # In this example the TLS key/cert were previously generated with the help of
+    # Let's Encrypt and placed in a dedicated directory into the daemon datadir
+    $ export TDEX_SSL_KEY=~/.tdex-daemon/trade-tls/privatekey.pem
+    $ export TDEX_SSL_CERT=~/.tdex-daemon/trade-tls/fullchain.pem
+
+    # Start a daemon with TLS enabled ONLY on Trade interface
+    $ tdexd
+    ```
+- Docker:
+    ```sh
+    # Start a dockerized daemon with TLS enabled ONLY on Trade interface
+    $ docker run -it -d \
+    --name tdexd \
+    --restart unless-stopped \
+    -p 9945:9945 -p 9000:9000 \
+    -v `pwd`/tdexd:/.tdex-daemon \
+    # Here, it's required to mount the filepaths of the TLS key/cert as volumes
+    # of the container
+    -v `pwd`/privatekey.pem:/privatekey.pem \
+    -v `pwd`/fullchain.pem:/fullchain.pem \
+    -e TDEX_SSL_KEY=/privatekey.pem \
+    -e TDEX_SSL_CERT=/fullchain.pem \
+    ghcr.io/tdex-network/tdexd:latest
+    ```
+
+### Enable Onion for Trade interface
+
+If your daemon remote host uses a linux system, instead of enabling TLS on the Trade interface, you can alternatively run a Tor hidden service acting as a proxy in front of it.
+
+If you opted for a dockerized solution, doing so if as simple as starting a container:
+
+```sh
+# If you already have an onion key you can use it by exporting env var,
+# otherwise just skip this step.
+$ export ONION_KEY=<your_oniion_key>
+
+# Start a tor hidden service that proxies incoming traffic on HTTP port to
+# daemon Trade interface on port 9945.
+$ docker -d \
+    run --name tor \
+    --network <network_with_tdexd_running>
+    --restart unless_stopped \
+    -e TDEX_TOR_SERVICE_HOSTS="80:tdexd:9945" \
+    -e TDEX_TOR_SERVICE_VERSION="3" \
+    -e TDEX_TOR_SERVICE_KEY=${ONION_KEY} \
+    goldy/tor-hidden-service:latest
+```
+:::tip
+To easily orchestrate multiple containers inter-connected to each other, you might find **[TDex Box](https://github.com/tdex-network/tdex-box)** as an interesting solution for this purpose. Rather than having to run each container manually, it simplifies the process to editing a YAML configuration file and running a single command to spin up multiple containers (it is essentially a [docker-compose](https://docs.docker.com/compose/) file).
```

Comment:
```suggestion
To easily orchestrate multiple containers inter-connected to each other, you might find **[TDEX Box](https://github.com/tdex-network/tdex-box)** as an interesting solution for this purpose. Rather than having to run each container manually, it simplifies the process to editing a YAML configuration file and running a single command to spin up multiple containers (it is essentially a [docker-compose](https://docs.docker.com/compose/) file).
```